pkg/deployer/terraform/deployer.go

```diff
@@ -396,7 +401,7 @@ func (d Deployer) createRevision(
 		return nil, err
 	}
 
-	if prevEntity != nil && prevEntity.Status == status.ServiceRevisionStatusRunning {
+	if prevEntity != nil && status.ServiceRevisionStatusReady.IsUnknown(prevEntity) {
```

Comment:
Previously, the if condition is equals to a running status. Here, the state is ready

Reply:
The unknown ServiceRevisionStatusReady condition is stand for the service revision is created but not finished, it means the "running state" as before. More conditions or status has more precise meaning. It's explained below.